WBooks/Navigation Bar/NavBarButtons.swift

```diff
@@ -22,4 +22,16 @@ extension UIBarButtonItem {
         searchButton.frame = CGRect(x: 0, y: 0, width: 34, height: 34)
         return UIBarButtonItem(customView: searchButton)
     }
+    
+    static func backNavBarButton(controller: UIViewController) -> UIBarButtonItem {
+        let backButton = UIButton(type: .system)
+        backButton.setImage(#imageLiteral(resourceName: "ic_back").withRenderingMode(.alwaysOriginal), for: .normal)
+        backButton.frame = CGRect(x: 0, y: 0, width: 34, height: 34)
+        backButton.reactive.controlEvents(.touchUpInside).observeValues { _ in
+            if let currentNavigationController = controller.navigationController {
```

Comment:
It would be better to get a closure as param for this function, so the action is defined by the controller that uses it, and not by the button it self